poglink/models/rates.py

```diff
@@ -141,7 +142,7 @@ def to_embed(self):
         # rename keys and format embed description
         embed_description = "\n".join(
             [
-                f"{v.rstrip('.0')} × {self.old.RATES_NAMES.get(k,k)}"  # TODO: rstrip won't remove '.0' if wrapped in ** from above
+                "{} × {}".format(re.sub(r"\.0", "", v), self.old.RATES_NAMES.get(k, k))
```

Comment:
[Can't use backslashes in f-strings](https://www.python.org/dev/peps/pep-0498/#:~:text=Backslashes%20may%20not%20appear%20inside%20the%20expression%20portions%20of%20f%2Dstrings%2C%20so%20you%20cannot%20use%20them%2C%20for%20example%2C%20to%20escape%20quotes%20inside%20f%2Dstrings%3A) so I used a different syntax to apply the formatting.

Needed to escape the `.` because in regex, that means "any character"